students/16th/heejinkim/js/login.js

```diff
@@ -0,0 +1,14 @@
+const loginId = document.querySelector(".js-id"),
+    loginPw = document.querySelector(".js-pw"),
+    loginBtn = document.querySelector(".js-btn");
+
+function changeColor() {
+    if (loginId.value !== "" && loginPw.value !== "") {
+        loginBtn.classList.add("changecolor");
+    } else {
+        loginBtn.classList.remove("changecolor");
+    }
```

Comment:
value가 빈 스트링이 아니라는 건 값이 있다는 것과 같으므로 다음과 같이 줄일 수 있습니다!
```
    if (loginId.value && loginPw.value) {
```
그리고 삼항연산자를 이용하면 다음처럼도 사용할 수 있겠네요!
```suggestion
const isValid = loginId.value && loginPw.value;
isValid ? loginBtn.classList.add("changecolor") : loginBtn.classList.remove("changecolor");
```